src/modules/Dashboard/features/Applet/Popups/TransferOwnershipPopup/TransferOwnershipPopup.tsx

```diff
@@ -1,12 +1,11 @@
-import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import { Modal } from 'shared/components';
 import { popups, applet } from 'redux/modules';
 import { useAppDispatch } from 'redux/store';
 import { TransferOwnership } from 'modules/Dashboard/features/Applet/TransferOwnership';
 import { StyledModalWrapper } from 'shared/styles/styledComponents';
-import { Mixpanel } from 'shared/utils';
+import { useTransferOwnership } from 'shared/hooks/useTransferOwnership';
```

Comment:
why not just from 'shared/hooks'?

Reply:
It will reduce potential circular dependencies were fixed earlier because of `shared/hooks`, `shared/utils`, `shared/components` and similar imports.